.github/workflows/cmake.yml

```diff
@@ -0,0 +1,37 @@
+name: CMake
+
+on:
+  push:
+    branches: [ "2.2.2-yb" ]
```

Comment:
Replace with `${{ github.event.repository.default_branch }}` so we don't have to change this each time we update gflags.

Reply:
Looks like that is not possible
https://stackoverflow.com/questions/64781462/github-actions-default-branch-variable